addons/addon-base-raas/packages/base-raas-services/lib/environment/environment-mount-service.js

```diff
@@ -667,17 +667,50 @@ class EnvironmentMountService extends Service {
     return policyDoc.Statement;
   }
 
+  /**
+   * This method looks for inline policy based on the given array of "possibleInlinePolicyNames".
+   * The method returns as soon as it finds an inline policy in the given role (identified by the "roleName")
+   * with a matching name from the "possibleInlinePolicyNames".
+   * If no inline policy is found with any of the names from the "possibleInlinePolicyNames", the method returns undefined.
+   *
+   * @param {Object} possibleInlinePolicyNames - Known possible policy role names we have used for out of the box CfN templates
```

Comment:
nit: `Known possible policy role names we have used for out of the box CfN templates` seems incorrect. This arg is not about role names, it's possible inline policy names (and not role names).

Reply:
Ah, that's right. Good catch